TestUtilities/Helpers/TestsDirectory.swift

```diff
@@ -0,0 +1,49 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import Foundation
+import XCTest
+
+/// Creates `Directory` pointing to unique subfolder in `/var/folders/`.
+/// Does not create the subfolder - it must be later created with `.create()`.
+/// It returns different `Directory` each time it is called.
+public func obtainUniqueTemporaryDirectory() -> URL {
```

Comment:
The temporary directory has been moved to `TestUtilities` but without the dependency to `Directory` or `File` from the v1 module. By the end of the migration I'm hoping that this logic will be used by the IntegrationTests only.